scst-store/amr/data-model-and-concepts.hbs.md

```diff
@@ -0,0 +1,31 @@
+# AMR Data Model and Concepts
+
+This topic describes data models currently used in the Supply Chain Security Tools (SCST) – Artifact Metadata Repository (AMR).
+
+## <a id='overview'></a> Overview
+
+Here is a diagram that shows the data models currently used in AMR to store artifact information and the relation between them.
+
+![Diagram of the AMR Data Models](../images/AMR-data-model.jpg)
+
+## AMR Data Models
+
+Currently in the AMR there are two types of data models, `App` and `Location`.
```

Comment:
⛏️ Since this is the first time you are mentioning the short form "App", you should use "Application" instead as that is also the name of the section.

Reply:
Oh good catch! this the term we should be using is "Apps" since that is what it's called in the graphql query, I will change the section name from `Application` to `Apps` and double check anywhere that `Apps` should be used